plugins/modules/info.py

```diff
@@ -2047,6 +2133,21 @@ def __init__(self):
         }
         LOG.info('Got Py4ps connection object %s', self.conn)
 
+    def get_acl(self, smb_share_id):
+        """
+        Retrieves the access control list (ACL) for a given SMB share ID.
+        Parameters:
+            smb_share_id (str): The ID of the SMB share.
+        Returns:
+            The access control list (ACL) for the specified SMB share ID.
+        """
+        try:
+            acl_response = self.provisioning.get_acl(smb_share_id)
+            acl_response = acl_response["aces"]
```

Comment:
I guess it should be access ?
```suggestion
            acl_response = acl_response["access"]
```

Reply:
this is returned by SDK response, so the keyword cannot be changed